.circleci/config.yml

```diff
@@ -66,6 +66,14 @@ jobs:
     <<: *job_common
     environment:
       - NODE_ENV: "test"
+      - IDBROKER_BASE_URL: "https://idbrokerbts.webex.com"
```

Comment:
Are WHISTLER and HYDRA needed?

Reply:
We aren't testing anything with whistler yet. I'll add hydra, although tests are passing without it somehow